spec.md

```diff
@@ -422,9 +422,10 @@ reference may include a tag or digest.
 
 The client should include an Accept header indicating which manifest content
 types it supports. For more details on the manifest formats and their content
-types, see [manifest-v2-1.md](manifest-v2-1.md) and
-[manifest-v2-2.md](manifest-v2-2.md). In a successful response, the Content-Type
-header will indicate which manifest type is being returned.
+types, see [manifest-v2-1.md](https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-1.md)
+and [manifest-v2-2.md](https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-2.md).
```

Comment:
If these links stay (and aren't replaced with, say, image-spec links or something more generic), you might want to pin to a particular docker/distribution tag or commit instead of floating them with `master`.

Reply:
not sure right now. but we can go with master for now.